lib/plugins/aws/provider.js

```diff
@@ -781,6 +781,7 @@ class AwsProvider {
                 name: { type: 'string' },
                 payload: { type: 'string' },
                 metrics: { type: 'boolean' },
+                useProviderTags: { type: 'boolean', enum: [true] },
```

Comment:
Let's use `{ const: true }` here

Reply:
updated it. Just out of curiosity 
this was the warning message displayed in my local
`Configuration warning at 'provider.httpApi.useProviderTags': should be equal to constant`
it does not specify which const (even what enum) value to specify